### ISSUE TITLE
Inconsistent property name of value argument

### ISSUE THREAD

**Issue by shanretoo** [substrait-io/substrait#639]
The documentation specifies the property name for the type associated with the value argument as `Type`, yet in the codebase, this property is actually named `value`. This inconsistency could lead to confusion.

https://github.com/substrait-io/substrait/blob/3dc77aeca820eba70ba141bf3afa5e4c5ba055b9/site/docs/expressions/scalar_functions.md#L27-L32

https://github.com/substrait-io/substrait/blob/3dc77aeca820eba70ba141bf3afa5e4c5ba055b9/text/simple_extensions_schema.yaml#L114-L122

In [substrait-java](https://github.com/substrait-io/substrait-java/blob/79decd20e85d6a1a5623890042ebcf1415cf784a/core/src/main/java/io/substrait/extension/SimpleExtension.java#L102-L105):

```
  public abstract static class ValueArgument implements Argument {

    @JsonProperty(required = true)
    public abstract ParameterizedType value();
```

**Comment by shanretoo** [comment]
Also, the `description` property is missing in the doc.

**Comment by westonpace** [comment]
Yes, you are correct.  At this point, given we have a lot of YAML files already (and they are using `value`) I think we should change the site to match the YAML / schema (i.e. change `type` to `value` and add `description`).  Are you interested in creating a PR?

**Comment by shanretoo** [comment]
Sure, I'll draft a pr.

**Comment by shanretoo** [comment]
Fixed in #640.
